extensions/ipynb/src/notebookImagePaste.ts

```diff
@@ -62,8 +61,8 @@ class CopyPasteEditProvider implements vscode.DocumentPasteEditProvider {
 		// create a snippet for paste
 		const pasteSnippet = new vscode.SnippetString();
 		pasteSnippet.appendText('![');
-		pasteSnippet.appendPlaceholder(`${pasteFilename}`);
-		pasteSnippet.appendText(`](attachment:${pasteFilename})`);
+		pasteSnippet.appendPlaceholder(`${revisedFilename}`);
```

Comment:
Should the placeholder  continue to use the original file name? That would make more sense to me since the `-1`, `-2` likely isn't useful alt text